docs/observability.md

```diff
@@ -14,44 +14,45 @@ The instrumentation is **experimental** - volume and semantics of the telemetry
 
 To enable the instrumentation:
 
-1. Set instrumentation feature-flag using one of the following options:
-
-   - set the `OPENAI_EXPERIMENTAL_ENABLE_OPEN_TELEMETRY` environment variable to `"true"`
-   - set the `OpenAI.Experimental.EnableOpenTelemetry` context switch to true in your application code when application
-     is starting and before initializing any OpenAI clients. For example:
-
-     ```csharp
-     AppContext.SetSwitch("OpenAI.Experimental.EnableOpenTelemetry", true);
-     ```
-
-2. Enable OpenAI telemetry:
+1. Enable experimental OpenAI telemetry:
 
    ```csharp
    builder.Services.AddOpenTelemetry()
        .WithTracing(b =>
        {
-           b.AddSource("OpenAI.*")
+           b.AddSource("Experimental.OpenAI.*", "OpenAI.*")
```

Comment:
nit: I think its best to mention in comment that "if you want to avoid changing code when things move to stable, add the m both now.", instead of showing them.

Reply:
Added some info below where I also explain what happens on these lines